labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -37,7 +41,10 @@ object RecursionPatternMatchingExercise {
    * List(1,1,2,3,1,1) -> List(1,1), List(2), List(3), List(1,1)
    */
   def groupConsecutive[T](in: List[T]): List[List[T]] = {
-    error("fix me")
+    in match {
+      case Nil => Nil
+      case a :: _ => (in takeWhile (_ == a)) :: groupConsecutive(in dropWhile (_ == a))
+    }
   }
```

Comment:
I wonder what happens if you don't have the Nil case, since it seems like most people left that out.